packages/discovery-react-components/src/components/StructuredQuery/__tests__/StructuredQuery.test.tsx

```diff
@@ -90,23 +174,18 @@ describe('<StructuredQuery />', () => {
             <StructuredQuery
               messages={{
                 addRuleGroupText: 'A new rules group',
-                fieldDropdownPlaceholderText: 'Field choice selection',
```

Comment:
why is this being removed? shouldn't the placeholder still exist until a selection is made?

Reply:
I originally removed this because in this case we have not setup our DummySearchClient to return fields, thus the loading text should be displayed instead of the placeholder text. I've since updated this test to check for the existence of the loading text instead of just removing the placeholder text